.github/workflows/integ_test.yml

```diff
@@ -112,7 +112,6 @@ jobs:
           scheme: GraphQLWithIAMIntegrationTests
 
   api-graphqluserpool-integration-test:
-    if: ${{ false }}
```

Comment:
removing this expects the test to pass and it's test result contributes to the overall success of the integration test workflow